### PR TITLE
PWA tutorial->CycleTracker

### PR DESCRIPTION
**Pull request by wbamberg**
This PR renames the CycleTracker tutorial from "Intro" to "CycleTracker", changes title prefixes from "PWA tutorial" to "CycleTracker:" and adds `short-title` to omit the prefix in sidebar and breadcrumbs.

With this PR and the corresponding change in the sidebar macro, the sidebar will look like:

Screen Shot 2023-06-29 at 12 09 11 PM

### CODE REVIEW

File: files/en-us/web/progressive_web_apps/tutorials/cycletracker/index.md [hunk 1]
```diff
@@ -38,39 +38,39 @@ Once installed, PWAs can be made to appear and act similarly to other applicatio
...
+- [Manifest: identity, appearance, and iconography](/en-US/docs/Web/Progressive_web_apps/Tutorials/CycleTracker/Manifest_file)
 
   - : A PWA requires a manifest, which is a JSON file that describes the name, icon, description, and other definitions of how the application works within the operating system on which the PWA is installed. We will create a manifest file that defines the appearance of the application when installed, including which icons should be used depending on the user's device, and parameters for the application viewport. We also look at debugging the manifest file with browser developer tools.
 
-- [Service workers](/en-US/docs/Web/Progressive_web_apps/Tutorials/Intro/Service_workers)
+- [Service workers](/en-US/docs/Web/Progressive_web_apps/Tutorials/CycleTracker/Service_workers)
 
   - : The service worker enables the application to work offline. With the secure connection in the previous step, the initial visit to a page provides its baseline functionality while the service worker downloads. After a service worker is installed and activated, it controls the page to offer improved reliability and speed.
```
Comment (bsmth) [inline files/en-us/web/progressive_web_apps/tutorials/cycletracker/index.md:75]: There's another two in the comments below (L79-83), maybe we want to get those also?

File: files/en-us/web/progressive_web_apps/tutorials/index.md [hunk 2]
```diff
@@ -7,7 +7,7 @@ slug: Web/Progressive_web_apps/Tutorials
 
 This page lists tutorials to help learn how to develop PWAs. Tutorials walk through the steps of creating an app, from start to finish, explaining how the different features of the app are implemented.
 
-- [Creating your first PWA](/en-US/docs/Web/Progressive_web_apps/Tutorials/Intro)
+- [Creating your first PWA](/en-US/docs/Web/Progressive_web_apps/Tutorials/CycleTracker)
```
Comment (bsmth) [inline files/en-us/web/progressive_web_apps/tutorials/index.md:10]: There's another straggler in `files/en-us/web/progressive_web_apps/index.md`:

https://github.com/mdn/content/blob/02c12a4ff416ec1c769ba536ce92b50c59c1b3e5/files/en-us/web/progressive_web_apps/index.md?plain=1#L54

**Review by bsmth — APPROVED** [review]
Thanks, Will, there's two places left to update, but feel free to merge after 👍🏻

**Comment by bsmth** [comment]
Relates to https://github.com/mdn/yari/pull/9192

**Comment by wbamberg** [comment]
Thank you for catching those! I shoulda grepped 😆

**Pull request merged**